src/constraints/util/isUnique.ts

```diff
@@ -0,0 +1,8 @@
+import fastDeepEqual from 'fast-deep-equal';
```

Comment:
We probably wanna use the /es6 export instead, as it also handles maps, sets, etc

Reply:
done